instances/atlas/parcellationEntity/JBA/JBA_cingulate-gyrus--frontal-part.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_cingulate-gyrus--frontal-part",
```

Comment:
special signs such as "," are typically replaced by "_" in most namings we use

Reply:
I don't see `,` ? do you mean `--` ?

Reply:
The name of "JBA_cingulate-gyrus--frontal-part" is "cingulate-gyrus, frontal-part".
Typically we replace special signs with "_" in the "@id" and file names. E.g.  "JBA_cingulate-gyrus_frontal-part"

But: I realized that in all other cases I think in the JBA uses the convention "frontal ...", "dorsal ..." etc. 
Should we just rename this completely anyways? To "JBA_frontal-cingulate-gyrus"?

Reply:
Yes. I think this is done in nearly all cases (e.g., "lateral occipital cortex", "frontal pole", etc)